ProcessLib/RichardsMechanics/RichardsMechanicsFEM.h

```diff
@@ -65,10 +65,12 @@ class RichardsMechanicsLocalAssembler
                              ShapeMatricesTypePressure, DisplacementDim,
                              ShapeFunctionDisplacement::NPOINTS>;
 
-    static int const KelvinVectorSize =
+    static constexpr int const KelvinVectorSize =
```

Comment:
either const or constexpr, but both are not necessary, especially the static const is already a constant expression. If it wouldn't be, one could not use it as a template argument below.